x-pack/plugins/observability/public/components/app/section/uptime/index.tsx

```diff
@@ -27,13 +27,14 @@ import { useChartTheme } from '../../../../hooks/use_chart_theme';
 import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
 import { useHasData } from '../../../../hooks/use_has_data';
 import { useTimeRange } from '../../../../hooks/use_time_range';
+import { BucketSize } from '../../../../pages/overview';
 import { Series } from '../../../../typings';
 import { ChartContainer } from '../../chart_container';
 import { StyledStat } from '../../styled_stat';
 import { onBrushEnd } from '../helper';
 
 interface Props {
-  bucketSize?: string;
+  bucketSize: BucketSize;
```

Comment:
Isn't this just a `number`?
```suggestion
  bucketSize: number;
```

Reply:
it was, now we need both the buckeSize and the intervalString, which we got by calling `getBucketSize` function.